dashboard/app/dsl/level_group_dsl.rb

```diff
@@ -28,6 +29,22 @@ def page
     @hash[:pages] << {levels: @current_page_level_names}
   end
 
+  def text(name)
+    # Ensure level is appropriate type.
+    level = Level.where(name: name).first # For some reason find_by_name doesn't always work here!
```

Comment:
Do we have any understanding as to why `find_by_name` doesn't always work? Is it safe to change this to use `Script.cache_find_level` (a new method that I'm adding)?